docs/docsite/rst/dev_guide/developing_collections_path.rst

```diff
@@ -0,0 +1,160 @@
+
+You can install collections made by others or share yours with the community through a distribution server such as Ansible Galaxy. Certified collections can be published to the Red Hat Automation Hub, a part of the Red Hat Ansible Automation Platform.
+
+Creating and sharing collections is a great way of contribution to the Ansible project.
+
+The Ansible community package consists of ``ansible-core``, which, among other core components, includes the ``ansible.builtin`` collection maintained by the Core team, and a set of collections maintained by the community.
+
+The purpose of this guide is to give your as a (potential) content creator a consistent overview of the Ansible collection creator journey from an idea for the first module/role to having your collection included in the Ansible community package. The :ref:`Collection development guidelines section<developing_collections>` provides references to more detailed aspect-specific documents.
+
+Each of the following guide sub-sections will reflect one milestone in the path.
+
+.. _examine_existing_content:
+
+Examine currently available solutions
+=====================================
+
+If you have an idea for a new role or module/plugin, there is no need to reinvent the wheel if there is already a sufficient solution that solve your automation issue.
+
+Therefore, first examine the currently available content including:
+
+* `Ansible builtin modules and plugins <https://docs.ansible.com/ansible/latest/collections/ansible/builtin/index.html>`_
+* :ref:`Ansible package collection index<all_modules_and_plugins>`
+* `Ansible Galaxy <https://galaxy.ansible.com/>`_
+* `Ansible Automation Hub <https://www.ansible.com/products/automation-hub>`_ if you have the Ansible Automation Platform subscription
+
+In case the solutions you found are not fully sufficient or have flaws, consider improving them rather than creating your own.
+
+If you already have your content written and used in your workflows, you could think of integrating it to the existing solutions.
+However, if they have significant flaws, licensing or major design issues, their corresponding projects are unmaintained, or if you think your implementation will work better, you are encouraged to create and share your own one.
+
+.. _create_content:
+
+Create your content
+===================
+
+You :ref:`tried <examine_existing_content>` but have not found any sufficient solution for your automation issue.
+
+Use one of the following guides:
+
+* :ref:`Roles guide<playbooks_reuse_roles>`: if you want to create a role.
+* :ref:`Developer guide<developer_guide>`: if you want to create a new Ansible module or plugin.
+
+Put your content in a collection
+================================
+
+You :ref:`created <create_content>` new content.
+
+Now it is time to create a collection to share your work with the community.
```

Comment:
I'm not sure this is the next step. Collections still have use for personal/individual company use that is not shared with community. Consider adapting this to say something like "Now that you have a module, role, or playbook, consider if you want to group related content together in one reusable, sharable collection. This allows you to group new modules with sample playbooks or roles, or plugins together.

Reply:
Technically, it's OK to create a collection that has only one entity, like a module or a role.
I would rather emphasize reusability and sharing here (with the community, colleagues or customers).
Sharing should be their default behavior:)